### ISSUE TITLE
use case challenge-tell-me-about-yourself produces incomplete json and database records

### ISSUE THREAD

**Issue by MichaelKentBurns** [MichaelKentBurns/MKB-cohort-1-Demo#20]

**Comment by bahati7** [comment]
Hello Michael, would you please give more information about this issue?

**Comment by MichaelKentBurns** [comment]
Yes.  I’m almost done collecting the info. Sent from my iPadOn Jul 3, 2024, at 13:48, bahati7 ***@***.***> wrote:﻿
Hello Michael, would you please give more information about this issue?

**Comment by MichaelKentBurns** [comment]
Never mind.  It was a testing mixup on my part.
I have stored the test and the results in 
test/use-cases/challenge-tell-me-about-yourself
The survey is the .json file.
My preliminary test is in the folder 'Issue #20...'   but in reviewing the results I realized that I had three different runs.  The first one failed to store at all because I was running it in my private XAMPP and survey.db has permission issues.  
The rest is in 'Retest' and includes the screen shots including the access_log entries and finally a screenshot of the records captured in the survey.db file copied there. 

There is one little issue in that the downloaded .json file has a name of 'unknown...' rather than 'Michael...'.   There might be an issue there. 

Marking this issue as closed because it was a testing mishap.   use case works as expected.
